### PR TITLE
Added Fullscreen and Unfullscreen Buttons in Game of Life Activity

### PR DESCRIPTION
**Pull request by jontymagan**
Added Icons and updated activity.js

### CODE REVIEW

File: activities/GameOfLife.activity/js/activity.js [hunk 1]
```diff
@@ -217,6 +217,26 @@ function main(Board, State, patterns, color, shadeColor, l10n, dataStore, genSpe
     });
   })
 
+  function resize() {
```
Comment (llaske) [inline activities/GameOfLife.activity/js/activity.js]: I don't think you need to reinvent a resize function. There is an existing one [here](https://github.com/llaske/sugarizer/blob/master/activities/GameOfLife.activity/js/activity.js#L208) that you could probably call when clicking on fullscreen/unfullscreen buttons.

Comment (jontymagan) [reply activities/GameOfLife.activity/js/activity.js]: I guess the resize function is not effective (the inbuilt one)(handleResize). I removed it and then tested. It is responsible for responsiveness and the handleResize function has a fixed width and length of the Canvas and it changes the size of the canvas .But we have to change the size of the main div to make the activity responsive on fullscreen which is possible only if we change the size of the main division which I did in my own function....
So I guess having a function will be good....

Comment (jontymagan) [reply activities/GameOfLife.activity/js/activity.js]: What are your thoughts?

Comment (llaske) [reply activities/GameOfLife.activity/js/activity.js]: The handleResize works, try to resize the window, you will see that the method is called each time and the board is resized. So you have to try to call it in the same way.

**Pull request merged**